vendor/update-pp.sh

```diff
@@ -20,6 +21,8 @@ mkdir -p pp/src
 SRC=$TMP/pp
 
 cp -v $SRC/src/pp.{ml,mli} pp/src
+cp -v $SRC/test/tests.ml pp/test/pp_tests.ml
```

Comment:
why do we rename the tests? seems like a gratuitous change.

Reply:
I wanted to preserve the name from before, but I can get rid of it.

Reply:
I say we should stay as close to upstream as possible